src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,32 @@
 import React from 'react';
+import './CommentInfo.scss';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+interface Props {
+  comment: Comment;
+}
+
+export const CommentInfo: React.FC<Props> = (props) => {
+  const { comment } = props;
```

Comment:
it's usually done like this 
```suggestion
export const CommentInfo: React.FC<Props> = ({comment}) => {
```